lib/mix/tasks/phx.gen.live.ex

```diff
@@ -219,6 +219,31 @@ defmodule Mix.Tasks.Phx.Gen.Live do
       """
     end
     if context.generate?, do: Gen.Context.print_shell_instructions(context)
+    maybe_print_upgrade_info()
+  end
+
+  defp maybe_print_upgrade_info do
+    unless Code.ensure_loaded?(Phoenix.LiveView.JS) do
+      Mix.shell().info """
+
+      Note the generated code requires Phoenix.LiveView v0.17+.
+
+      Before proceeding, update your :phoenix_live_view and,
+      if present, :phoenix_live_dashboard deps in mix.exs:
+
+          def deps do
+            [
+              {:phoenix_live_view, "~> 0.17.0"},
```

Comment:
This will get outdated. :( Should we say "we require Phoenix.LiveView v0.17+ and Phoenix.LiveDashboard v0.6+" and leave at that? We could also specify said requirements but maybe too broad?

Reply:
Oops I forgot to leave a note saying to reword as you see fit :)

> This will get outdated

I was concerned about that, too :(